### PR TITLE
Controllers refactor

### PR DESCRIPTION
**Pull request by ajmunoz411**
Sorry for this huge one. There's likely some optimization/tidying up to be done on my end but everything should be functioning properly
- Controllers.js deleted
- Controller functions split into sub-files
- Routes/controllers added

### CODE REVIEW

**Review by jeremysylee — APPROVED** [review]
Killing it, nice work.

File: server/controllers/coordinates.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const axios = require('axios');
+
+const getCoordinates = async (addressQuery) => {
```
Comment (raphaelspies) [inline server/controllers/coordinates.js:3]: Recommend deprecating API key and relocating new one to .env file

**Review by raphaelspies — CHANGES_REQUESTED** [review]
Looks great! MAKE SURE YOU REMOVE THE PARCEL FILES in dist from PR before merging.

File: server/controllers/coordinates.js (hunk 1 above)
Comment (ajmunoz411) [reply server/controllers/coordinates.js:3]: good call on parcel files. So I can delete that whole dist file correct?
and yea the API stuff is through Miguel, we'll have to get him to generate a new one

**Pull request merged**